x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curation_queries/curation_queries.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import { EuiButton, EuiButtonEmpty, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { Curation } from '../../types';
+
+import { CurationQueriesLogic } from './curation_queries_logic';
+import { CurationQuery } from './curation_query';
+import { filterEmptyQueries } from './utils';
+import './curation_queries.scss';
```

Comment:
You include this in 2 different components.

Reply:
Doh, awesome catch

Reply:
https://github.com/elastic/kibana/pull/92560/commits/3f70ef8121c4d3ee142f0c252d0a7ff167d794fd